client/src/views/Dashboard/SummaryContainer/TrendSummary/index.js

```diff
@@ -330,7 +332,12 @@ class TrendSummary extends React.PureComponent {
             <div className={_cs(styles.trend, className)}>
                 <div className={_cs(styles.item)}>
                     <h3> RC Supply Trend </h3>
-                    <GroupedBarChart
+                    {rcData.values.length > 0 && (
```

Comment:
Are you sure rcData.values is defined?
```suggestion
                    {rcData && rcData.values && rcData.values.length > 0 && (
```

Reply:
Same for the other ones.
If rcData is surely defined, then just check for rcData.values